src/include/tensor/hip_tensor_executors.hpp

```diff
@@ -366,14 +366,15 @@ RppStatus hip_exec_copy_tensor(T *srcPtr,
                                RpptDescPtr dstDescPtr,
                                rpp::Handle& handle);
 
-// -------------------- swap_channels --------------------
+// -------------------- channel_permute --------------------
 
 template <typename T>
-RppStatus hip_exec_swap_channels_tensor(T *srcPtr,
-                                        RpptDescPtr srcDescPtr,
-                                        T *dstPtr,
-                                        RpptDescPtr dstDescPtr,
-                                        rpp::Handle& handle);
+RppStatus hip_exec_channel_permute_tensor(T *srcPtr,
+                                          RpptDescPtr srcDescPtr,
+                                          T *dstPtr,
+                                          RpptDescPtr dstDescPtr,
+                                          Rpp32u *permutationsList ,
```

Comment:
I think we need to give permute order directly without having to use a map table to figure it out. For e.g, here you can give another tensor (2D array) pointer with permute order for the batch

Reply:
done